nifi-nar-bundles/nifi-soap-bundle/nifi-soap-processors/src/test/java/org/apache/nifi/processors/soap/GetSOAPIT.java

```diff
@@ -0,0 +1,146 @@
+package org.apache.nifi.processors.soap;
+
+import org.apache.nifi.util.MockFlowFile;
+import org.apache.nifi.util.TestRunner;
+import org.apache.nifi.util.TestRunners;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static org.apache.nifi.processors.soap.GetSOAP.REL_SUCCESS;
+
+/**
+ * Integration test that will query live soap services.
+ * Created by frankm on 2/28/17.
```

Comment:
what do you mean? it's in src/test/java which is standard. where else would it go?